ProcessMaker/screen-builder
# FOUR-16958: Regression - Watchers running scripts to clean data stops working and keeps the data

## Issue & Reproduction Steps

Expected behavior: 
The watcher and script to clean data should clean/update the data successfully without returning the old values.
Actual behavior: 
Watchers that are used to clean the data of fields using script tasks are not working anymore when running the watcher is cleaning the data but when you try to type a different one in the field it returns the old data, this happens in text area, input fields.
## Solution
- clean debouncedValuesQueue because this array acts like a cache

https://github.com/ProcessMaker/screen-builder/assets/1401911/96247730-6019-4500-9079-975073be3c60



## How to Test
Test the steps above

1. Import the following screen Reset this page.json 
2. Preview it or create a case
3. Use the button to clear the fields
4. Try to type again in the fields
5. At this point, you will see that the old data is returned to the fields.


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-16958

## Code Review Checklist
